packages/modules/viewer-react/src/components/error/ErrorBoundary.tsx

```diff
@@ -0,0 +1,43 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ErrorPage } from "@itwin/itwinui-react";
+import type { PropsWithChildren, ReactNode } from "react";
+import React, { Component } from "react";
+
+interface State {
+  fallback: boolean;
+  error: Error;
+}
+
+export class ErrorBoundary extends Component<
+  PropsWithChildren<ReactNode>,
```

Comment:
I believe this should just be `PropsWithChildren<{}>`

Reply:
Oh, whenever I use empty brackets pre-commit flags it:
15:64  error  Don't use `{}` as a type. `{}` actually means "any non-nullish value".
- If you want a type meaning "any object", you probably want `Record<string, unknown>` instead.
- If you want a type meaning "any value", you probably want `unknown` instead.
- If you want a type meaning "empty object", you probably want `Record<string, never>` instead  @typescript-eslint/ban-types

Reply:
I think we want to use `Record<string, never>` here

Reply:
I get different errors:
Overload 1 of 2, '(props: PropsWithChildren<Record<string, never>> | Readonly<PropsWithChildren<Record<string, never>>>): ErrorBoundary', gave the following error.       
    Type '{ children: Element; }' is not assignable to type 'Readonly<PropsWithChildren<Record<string, never>>>'.
      Property 'children' is incompatible with index signature.
        Type 'Element' is not assignable to type 'never'.

I'll try googling up the issue.

Reply:
hmm thats strange, try unknown instead of never then

Reply:
That works.